src/main/java/org/simulator/math/MaxDivergenceTolerance.java

```diff
@@ -0,0 +1,50 @@
+package org.simulator.math;
+
+import org.simulator.math.odes.MultiTable.Block.Column;
+
+public class MaxDivergenceTolerance extends QualityMeasure {
+
+    /**
+     * The metric the relative distance is based on
+     */
+    protected MaxAbsDistance metric;
+
+    /**
+     * Absolute tolerance for the reaction
+     */
+    protected double absTol;
+
+    /**
+     * Relative tolerance for the reaction
+     */
+    protected double relTol;
+
+    /**
+     * Default Constructor
+     */
+    public MaxDivergenceTolerance(double absTol, double relTol) {
+        super(Double.NaN);
+        metric = new MaxAbsDistance();
+        this.absTol = absTol;
+        this.relTol = relTol;
+    }
+
+    /* (non-Javadoc)
+     * @see org.sbml.simulator.math.Distance#distance(java.lang.Iterable, java.lang.Iterable, double)
+     */
+    @Override
+    public double distance(Column x, Column y, double defaultValue) {
```

Comment:
Also did not see any test code for this.

Reply:
Adding in the next commit!

Reply:
Added the tests for `MaxDivergenceTolerance` class